src/SayMore/Model/Files/FileType.cs

```diff
@@ -355,6 +355,7 @@ public class SessionFileType : FileType
 		public const string kGenreFieldName = "genre";
 		public const string kParticipantsFieldName = "participants";
 		public const string kTitleFieldName = "title";
+		public const string kContributionsFieldName = "contributions";
```

Comment:
Looks like there are some more places in the code where we should be using this constant.